deps/leader_cron-0.1/.gitignore

```diff
@@ -0,0 +1,11 @@
+erl_crash.dump
```

Comment:
On other note regarding gen_listener using AMQP.  I think that is very generic solution (for the project) that will benefit a number of applications (for example the fax worker).  If in our discussions that this the route we take that should become a core library.